__tests__/app.test.js

```diff
@@ -452,3 +452,37 @@ describe("10-GET/api/reviews?queries REFACTORS 4-GET:/api/reviews", () => {
       });
   });
 });
+
+describe("12-DELETE/api/comments/:comment_id", () => {
+  it("should delete the comment from the database responds with 204", () => {
+    return request(app)
+      .delete("/api/comments/2")
+      .expect(204)
+      .then(() => {
+        return db.query("SELECT * FROM comments").then((result) => {
+          const comments = result.rows;
+          comments.forEach((commentObj) => {
```

Comment:
Nice that you're testing the comments which come back, but remember if we're not checking the length of the array that comes back we could get a false positive from our tests.